### PR TITLE
Styling fixes

### PR DESCRIPTION
**Pull request by ganeshbsub**

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -7,7 +7,6 @@ body {
   font-weight: 300;
   font-family: 'Ubuntu', sans-serif;
   letter-spacing: 1px;
-  text-align: justify;
```
Comment (marcoow) [inline css/main.scss]: we should still justify text but limit it to `<p>` and maybe `<li>`

Comment (ganeshbsub) [inline css/main.scss]: Justify on `<li>` makes it look a weird in a couple of lines. Maybe just `<p>`?

Comment (marcoow) [inline css/main.scss]: yeah, not sure about `<li>` - if you say it looks weird we'll leave those left-aligned

**Comment by marcoow** [comment]
now the teasers on the top look weird:

bildschirmfoto 2016-08-11 um 17 53 41

Also the workshop format `<li>`s should be justified as well I think:

bildschirmfoto 2016-08-11 um 17 54 05

**Pull request merged**